Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,55 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string fullName = askFullName();
+            int age = askAge();
+            int weight = askWeight();
+            double height = askHeight();
+
+            Console.WriteLine($"{fullName} is {age} years old, his weight is {weight} kg and his height is {height:F1} cm.");
```

Comment:
doing the same for a second person is missing

Reply:
I wrote a loop around it so it will run twice.